CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/plugins/clean/RemoveNodesPluginNGTest.java

```diff
@@ -89,13 +90,33 @@ public void testCreateParameters() {
     @Test
     public void testEditAttributesNotFound() throws InterruptedException, PluginException {
```

Comment:
I think with the way this test has been modified, it would be better to split into 2 separate tests, one for selected missing and one for identifier missing. This would then allow you to make use of the expectedException parameter for the Test annotation instead of throwing fail if the exception doesn't occur. The 3rd case should be covered by the other tests (which in theory should run intended behaviour)